dev-requirements.txt

```diff
@@ -34,13 +34,13 @@ black==21.6b0
     # via dallinger
 bleach==3.3.0
     # via nbconvert
-boto3==1.17.105
+boto3==1.17.93
```

Comment:
Again, I'm curious why we're downgrading dependencies.

Reply:
Note to self: if you want all `.txt` files to be as _current_ master, this is the right command (after a git fetch):
```
git checkout origin/master -- *.txt
```
while this command will use a possibly outdated local version of `master`:
```
git checkout master -- *.txt
```